### PR TITLE
React capstone project (Crypto Currency)

### PR DESCRIPTION
**Pull request by sambeck87**
- Home page
  - When the page loads, the web app shows the list of categories that the category name could filter out.
  - Along with the category name, it will display some numeric values per category that come from the API.
  - When the user clicks (or taps) on a category item, the application navigates to the details page.

- Details page
  - In the details page, the web app retrieves data from the API to show detailed data in the category.
  - When the user clicks on the "Back" button (<), the user navigates to the home page.

- Testing features
  - This project contains unit tests for pure functions.
  - This project contains integration tests for your application using the React Testing Library.

- Technical features
  - The project is a single-page application (SPA) built with React and Redux.
  - The data retrieved from the API is stored in the Redux store.
  - It filters the data that is retrieved from the API using a Filter stateless component.
  - Every page (the main page and the pages for each item) have a unique route within the SPA.
  - The project was deployed and is accessible online

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
```
Comment (TedLivist) [inline README.md]: - [ ] Kindly fix this link. It should be `#deployment`.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @sambeck87,

Good job so far!
✔️ Well-written and passing tests and good documentation
✔️ Implementation of unique routes
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (sambeck87) [reply README.md]: Thank you, I fixed the problem

**Review by julie-ify — APPROVED** [review]
Hi @sambeck87,

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**